File: tmdb-core/src/main/java/dev/reuss/tmdb/value/image/size/ProfileSize.java
package dev.reuss.tmdb.value.image.size;

/**
 * Supported TMDB profile image sizes.
 */
public enum ProfileSize implements ImageSize {
    W45("w45"),
    W185("w185"),
    H632("h632"),
    ORIGINAL("original");

    private final String value;

    ProfileSize(String value) {
        this.value = value;
    }

    @Override
    public String value() {
        return value;
    }
}

File: tmdb-core/src/main/kotlin/dev/reuss/tmdb/value/image/size/ProfileSize.kt
package dev.reuss.tmdb.value.image.size

/** Profile-image size values exposed by [dev.reuss.tmdb.domain.images.url.ImageUrlBuilder]. */
enum class ProfileSize(
    override val value: String,
) : ImageSize {
    W45("w45"),
    W185("w185"),
    H632("h632"),
    ORIGINAL("original"),
}
